draft-irtf-panrg-path-properties.md

```diff
@@ -85,10 +85,10 @@ Note that this document does not assume that any of the listed path properties a
 # Terminology
 
 Entity:
-: A physical or virtual device or function which can process packets (i.e., a node or host), measure path properties, or access information about paths. With respect to a given communication (i.e., a flow), an entity may be on the data plane or control plane, and it may be on-path or off-path.
+: A physical or virtual device or function, or a collection of devices or functions, which can, for example, process packets, measure path properties, or access information about paths. With respect to a given communication, an entity may be on the data plane or control plane, and it may be on-path or off-path.
 
 Node:
-: An entity which processes packets, e.g., sends, receives, forwards, or modifies them. A node may be physical or virtual, e.g., a physical device or a service function provided as a virtual element. A node may also be the collection of multiple entities which, as a collection, processes packets, e.g., an entire Autonomous System (AS).
+: An entity which processes packets, e.g., sends, receives, forwards, or modifies them. A node may be physical or virtual, e.g., a physical device or a service function provided as a virtual element. A node may also be an entity which, as a collection, processes packets, e.g., an entire Autonomous System (AS).
```

Comment:
We should maybe clarify what the collection consists of:
"An entity which processes packets, e.g., sends, receives, forwards, or modifies them. A node may be physical or virtual, e.g., a physical device or a service function provided as a virtual element. A node may also be an entity which consists of a collection of devices or functions, e.g., an entire Autonomous System (AS)."

Reply:
That works! Then it doesn't say anymore that the collection processes packets, but that's both already part of the definition and clear from the example.